packages/react-lowcode/src/codegen/tests/list/list.test.ts

```diff
@@ -210,7 +210,47 @@ test(".grommet table generation without formatting", () => {
     let prindedSource = printer.printList(ts.ListFormat.MultiLine, factory.createNodeArray([...page!.imports, page!.functionDeclaration]), sourceFile)
 
     expect(prindedSource).toContain('<CustomerTable customers={data?.customers} />')
-});
+  });
+
+  describe("Generate page for list component", () => {
+    test("mui data table generation with formatting", () => {
+      const sourceFile = createAst('')
+      const myClassFile = parseGraphqlTypes(graphqlGenTs1)
+      const testEntity = sourceFileEntity(myClassFile)
+      const template = `
+import { useGeneratedQuery } from '../generated'
```

Comment:
Please let's test is with different templates (different control flow) to be sure that is more general enough.